src/routes/mentor/mentor.route.test.ts

```diff
@@ -60,6 +63,23 @@ describe('Mentor application', () => {
     it('should return a 401 when a valid access token is not provided', async () => {
       await supertest(server).post('/api/mentors').send({}).expect(401)
     })
+  })
+
+  describe('Get mentor details route', () => {
+    it('should return a 200 with the mentor details', async () => {
+      const response = await agent
+        .get(`/api/mentors/${savedMentor.uuid}`)
+        .expect(200)
+
+      expect(response.body).toHaveProperty('mentorId')
+      expect(response.body).toHaveProperty('category')
+      expect(response.body).toHaveProperty('profile')
+    })
```

Comment:
Write a test to call the API unauthenticated as well @Madhawa97

Reply:
Sure!